MovieGallery/Model/Movies.cs

```diff
@@ -80,10 +80,13 @@ internal class ParseStringConverter : JsonConverter
 
         public override object ReadJson(JsonReader reader, Type t, object existingValue, JsonSerializer serializer)
         {
-            if (reader.TokenType == JsonToken.Null) return null;
+            if (reader.TokenType == JsonToken.Null)
+            {
+                return null;
+            }
+
             var value = serializer.Deserialize<string>(reader);
-            long l;
-            if (Int64.TryParse(value, out l))
+            if (long.TryParse(value, out long l))
```

Comment:
Variable declaration can be inlined
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0018